dev/how-to/get-started/dm.md

```diff
@@ -0,0 +1,649 @@
+---
```

Comment:
Can we change the filename to `data-migration.md`? I think this is better for search engines + users.